apps/ledger-live-desktop/tests/page/swap.page.ts

```diff
@@ -197,6 +218,13 @@ export class SwapPage extends AppPage {
     await webview.getByRole("button", { name: `Swap with ${provider}` }).click();
   }
 
+  @step("Go to provider live app")
+  async goToProviderLiveApp(electronApp: ElectronApplication, provider: string) {
+    const [, webview] = electronApp.windows();
+    await expect(webview.getByRole("button", { name: `Continue with ${provider}` })).toBeVisible();
+    await webview.getByRole("button", { name: `Continue with ${provider}` }).click();
+  }
```

Comment:
Could we factorize `webview.getByRole("button", { name: 'Continue with ${provider}' }` a bit here ?